integrations/integration.php

```diff
@@ -78,33 +76,15 @@ public function __construct( string $slug ) {
 	 * @private
 	 */
 	public function activate( array $options = [] ): void {
-		// If integration is already available in customer code then don't activate it from platform side.
-		if ( $this->is_loaded() ) {
-			trigger_error( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
-				sprintf( 'Prevented activating of integration with slug "%s" because it is already loaded.', esc_html( $this->slug ) ),
-				E_USER_WARNING
-			);
-		}
-
-		// Don't do anything if integration is already activated.
-		if ( $this->is_active() ) {
-			trigger_error( sprintf( 'VIP Integration with slug "%s" is already activated.', esc_html( $this->get_slug() ) ), E_USER_WARNING ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+		// Don't do anything if integration is already loaded or activated.
+		if ( $this->is_loaded() || $this->is_active() ) {
+			return;
 		}
 
 		$this->is_active = true;
 		$this->options   = $options;
 	}
 
-	/**
-	 * Callback for `switch_blog` filter.
-	 */
-	public function switch_blog_callback(): void {
-		// Updating config to make sure `get_config()` returns config of current blog instead of main site.
-		if ( isset( $this->vip_config ) ) {
-			$this->options['config'] = $this->vip_config->get_site_config();
-		}
-	}
```

Comment:
This is no longer needed, because `get_network_site_config()` will always pull the config from the configmap based on the current network site's ID. It's not stored in this intermediary `$options['config']` anymore -- that is only used when integrations are manually activated w/ a custom config array.